src/Unity.Mathematics.PerformanceTests/TestHash.cs

```diff
@@ -0,0 +1,49 @@
+using Unity.Burst;
+using Unity.PerformanceTesting;
+using NUnit.Framework;
+using Unity.Collections.LowLevel.Unsafe;
+using System;
+using UnityEngine;
+
+namespace Unity.Mathematics.PerformanceTests
+{
+    [BurstCompile]
+    partial class TestHash
+    {
+        static readonly long kBufferBytes = 1024 * 1024;
+
+        [Test, Performance]
+        public static unsafe void HashAligned()
```

Comment:
Minor: but since these tests don't assert anything, it would be nice if they were named something like `HashAligned_ShouldNotCrash()` so it's clear that you didn't miss adding an assert and that just running is good enough (I mean, _most_ code should not crash as a minimum but here it's the only goal since no other qualities of the hash are being validated)